app/views/exchanges/hbx_profiles/_marketplace_plan_year.html.slim

```diff
@@ -0,0 +1,28 @@
+.top-pd.row#inbox_provider_form
```

Comment:
I'm a little curious what the `inbox_provider_form` does here. _Seems_ like it might be involved in hiding/showing sections with that id (maybe for a quick look at inbox contents?), but I'm just taking a guess. If no styling effect, maybe it can be omitted?